CHANGELOG.md

```diff
@@ -50,6 +50,11 @@ All notable changes to this project will be documented in this file.
     * #### Removed
 * ### `niscope` (NI-SCOPE)
     * #### Added
+        * Added methods for getting calibration information. - #1463
```

Comment:
Link to the issue from the issue#.

Reply:
>Talk about added functions in terms of their Python names. Remove word exposed. Remember this is read by Python users who should not need to know C APIs
When referring to an issue, don't paste full path, just write #1463.

From Marcos' review. Should I have the issue linked or not?

Reply:
In GitHub comments you can use the shortcut `#1463` and GitHub knows how to turn that into an actual link. In the CHANGELOG.md file, GitHub does not do that so links need to be explicit. You can see how they are formatted in markdown by looking at previous releases.